Malay-Kumar-Parida/02.Variables/main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import "fmt"
+
+var l float32 = 100 //lower case Thorughout This package
```

Comment:
```suggestion
var l float32 = 100 // Lower case throughout this package
```
when you comment give space after `// `.
Also, you can't use words starting small and capital letters you have to be consistent. Also, check typos when you comment.

do the similar changes in all other places.

Reply:
@malayparida2000 still needs to update